### ISSUE TITLE
data-node no longer runs 

### ISSUE THREAD

**Issue by gkowalski** [nlpsandbox/data-node#221]
### Problem description

RUnning the tagged 1.2.1 or latest main branches of the code fails on startup of the data-node

#### Steps to reproduce the issue
Checkout tag 1.2.1 

git checkout tags/1.2.1 -b 1.2.1
docker rm data-node data-node-db
docker volume prune 
docker-compose up --build 

```

data-node     | Traceback (most recent call last):
data-node     |   File "openapi_server/__main__.py", line 3, in <module>
data-node     |     import connexion
data-node     |   File "/usr/local/lib/python3.9/site-packages/connexion/__init__.py", line 14, in <module>
data-node     |     from .apis import AbstractAPI  # NOQA
data-node     |   File "/usr/local/lib/python3.9/site-packages/connexion/apis/__init__.py", line 16, in <module>
data-node     |     from .abstract import AbstractAPI  # NOQA
data-node     |   File "/usr/local/lib/python3.9/site-packages/connexion/apis/abstract.py", line 21, in <module>
data-node     |     from ..spec import Specification
data-node     |   File "/usr/local/lib/python3.9/site-packages/connexion/spec.py", line 11, in <module>
data-node     |     import jinja2
data-node     |   File "/usr/local/lib/python3.9/site-packages/jinja2/__init__.py", line 12, in <module>
data-node     |     from .environment import Environment
data-node     |   File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 25, in <module>
data-node     |     from .defaults import BLOCK_END_STRING
data-node     |   File "/usr/local/lib/python3.9/site-packages/jinja2/defaults.py", line 3, in <module>
data-node     |     from .filters import FILTERS as DEFAULT_FILTERS  # noqa: F401
data-node     |   File "/usr/local/lib/python3.9/site-packages/jinja2/filters.py", line 13, in <module>
data-node     |     from markupsafe import soft_unicode
data-node     | ImportError: cannot import name 'soft_unicode' from 'markupsafe' (/usr/local/lib/python3.9/site-packages/markupsafe/__init__.py)
data-node     | unable to load app 0 (mountpoint='') (callable not found or import error)
data-node     | *** no app loaded. going in full dynamic mode ***
data-node-db  |
data-node-db  | MongoDB init process complete; ready for start up.
data-node-db  |
```
 
This is due to removed dependency in the Markupsafe library ( https://markupsafe.palletsprojects.com/en/2.1.x/changes/#version-2-1-0 ) 

Fix is to update the ~/server/requirements.txt line to : 

connexion==2.11.2

**Comment by thomasyu888** [comment]
@gkowalski  Can you try to not have the `--build`?

**Comment by tschaffter** [comment]
See #222

**Comment by tschaffter** [comment]
This issue has been fixed via #222.
